### ISSUE TITLE
Reusable blocks: the "Manage all..." link in the Inserter should look like a link

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#10009]
When there are saved reusable blocks, a link that looks like a button is displayed in the inserter, in the Reusable block section:

![screen shot 2018-09-18 at 15 46 52](https://user-images.githubusercontent.com/1682452/45695617-11863f80-bb62-11e8-86c5-81ff9fbe0f96.png)

This is a link styled like a button. It just happened to me, using the keyboard, to press spacebar because it looks like a button and I've stared at the screen waiting for something to happen...
Of course, nothing happened because it's a link and it gets activated just with the Enter key.

This is a very good example of a design choice made just for visual purposes that is actually confusing for users. This is a link, it triggers navigation to an actual, different admin page (`wp-admin/edit.php?post_type=wp_block`) thus should look like a link.

Other improvements are certainly possible. For example, the discoverability of this control could be improved. Also, seems to me there's a bit of excessive title-casing: `Manage All Reusable Blocks` seems a bit too much to me.

For reference, see #8888 / #8927, and #4325 / #4471. See also the copy guidelines https://github.com/WordPress/gutenberg/blob/master/docs/reference/copy-guide.md#five-pay-attention-to-capitalization 

Seems there's still a bit of inconsistency across the UI:

<img width="766" alt="screen shot 2018-09-18 at 15 47 33" src="https://user-images.githubusercontent.com/1682452/45696924-e5b88900-bb64-11e8-8f56-205d67385e15.png">
<img width="704" alt="screen shot 2018-09-18 at 15 47 39" src="https://user-images.githubusercontent.com/1682452/45696925-e6511f80-bb64-11e8-9d22-0a098d4011f8.png">
<img width="714" alt="screen shot 2018-09-18 at 15 47 41" src="https://user-images.githubusercontent.com/1682452/45696926-e6511f80-bb64-11e8-89ae-a045b861b3c8.png">
<img width="731" alt="screen shot 2018-09-18 at 15 47 42" src="https://user-images.githubusercontent.com/1682452/45696927-e6511f80-bb64-11e8-8352-ca0061babb3b.png">

**Comment by afercia** [comment]
Worth noting in the "More" menu, it looks like a link (it also becomes blue on hover...)

<img width="377" alt="manage all rb" src="https://user-images.githubusercontent.com/1682452/45697359-eef62580-bb65-11e8-95bf-44536871dc8b.png">

Re: title casing, there are various inconsistencies here too, as only feature names should be title cased. Cases like `Copy All Content` are, honestly, a bit funny 🙂

**Comment by youknowriad** [comment]
Also see #9989

**Comment by karmatosed** [comment]
I want to just check I understand what is being suggested. The styling is not going to be changed - it will still be an icon - but it's how it's formatted?

**Comment by afercia** [comment]
> The styling is not going to be changed

Icon-only controls are an anti-pattern for accessibility and should be avoided when possible. The a11y feedback is to change this in a textual link that looks like a link. That said, if the team doesn't want to change it then it's a responsibility of the team if this control is not fully accessible. Thanks.

**Comment by tofumatt** [comment]
Chiming in here from: from both an accessibility and usability perspective this should not look like a button. It's a really janky UX and it looking like a button makes me think clicking it will keep me in the editor context, but once I do click it (or press `Enter`), I'm prompted with Gutenberg's on-page-leave dialog because it's changing the page:

![2018-10-09 18 02 57](https://user-images.githubusercontent.com/90871/46685572-bdfd9380-cbed-11e8-91cc-e4a3c3f12927.gif)

That is super-unexpected from just a UX perspective. Additionally, there *are* accessibility concerns because I too expect that button to… act like a button 😄

That "button" is a link to another page on the site entirely and should be communicated as such. I don't see why we can't do:

<img width="792" alt="screenshot 2018-10-09 18 09 19" src="https://user-images.githubusercontent.com/90871/46685868-74fa0f00-cbee-11e8-82b0-78a098a96977.png">

That much more clearly communicates the action of clicking on that item.

**Comment by karmatosed** [comment]
@tofumatt can we align to the right the link? It would fit in better I think with actions if that happened. Thanks for stepping in.

**Comment by tofumatt** [comment]
Works for me! I'll make the change in about an hour and update the screenshots.
